src/abstractdataframe/join.jl

```diff
@@ -555,14 +595,21 @@ the result. A left join includes all rows from `df1`.
   if `true`, duplicate names will be suffixed with `_i`
   (`i` starting at 1 for the first duplicate).
 - `indicator` : Default: `nothing`. If a `Symbol` or string, adds categorical indicator
-   column with the given name, for whether a row appeared in only `df1` (`"left_only"`),
-   only `df2` (`"right_only"`) or in both (`"both"`). If the name is already in use,
-   the column name will be modified if `makeunique=true`.
+  column with the given name, for whether a row appeared in only `df1` (`"left_only"`),
+  only `df2` (`"right_only"`) or in both (`"both"`). If the name is already in use,
+  the column name will be modified if `makeunique=true`.
 - `validate` : whether to check that columns passed as the `on` argument
-   define unique keys in each input data frame (according to `isequal`).
-   Can be a tuple or a pair, with the first element indicating whether to
-   run check for `df1` and the second element for `df2`.
-   By default no check is performed.
+  define unique keys in each input data frame (according to `isequal`).
+  Can be a tuple or a pair, with the first element indicating whether to
+  run check for `df1` and the second element for `df2`.
+  By default no check is performed.
+- `rename` : a `Pair` specifying how columns of left and right data frames should
+  be renamed in the resulting data frame. Each element of the pair can be a
+  string or a `Symbol` can be passed in which case it is appended to the original
+  column name; alternatively a function can be passed in which case it is applied
+  to each column name, which is passed to it as a `String`.; Note that `rename`
```

Comment:
Also need to fix this occurrence and the next ones.

Reply:
What is wrong here? I thought I have fixed all (I left the `.;` after `String` in these lines which I just pushed a fix for, but s there some problem with the wording?)

Reply:
I was referring to ;.